.github/workflows/FE_PR.yaml

```diff
@@ -77,8 +77,10 @@ jobs:
         working-directory: client
         run: npm run generate
 
+      - name: Unit Tests
+        working-directory: client
+        run: npm run test
```

Comment:
I know in the github example [here](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs) that they put the testing step after the build. Does that make a difference?